lib/resources/datav2/websocket.ts

```diff
@@ -2,6 +2,7 @@ import events from "events";
 import WebSocket from "ws";
 import { MessagePack } from "msgpack5";
 import msgpack5 from "msgpack5";
+import { timeStamp } from "console";
```

Comment:
[minor] Can you please tell me where/how this is used?

Reply:
Forgot to delete